pandas/core/indexes/datetimelike.py

```diff
@@ -97,6 +97,53 @@ class TimelikeOps(object):
         ValueError if the freq cannot be converted
         """)
 
+    _floor_doc = (
+        """
+        Floor the DatetimeIndex to the specified freq.
+
+        All the available freq can be found in the following link
+        in the offset-aliases section:
+        http://pandas.pydata.org/pandas-docs/stable/timeseries.html
```

Comment:
Consider using an rST reference and directly link to the section: http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases